api-authorizations/README.adoc

```diff
@@ -0,0 +1,78 @@
+Rudder plugin: API Authorizations
+---------------------------------
+
+This project is part of Rudder - Continuous configuration for effective compliance
+
+Rudder is an easy to use, web-driven, role-based solution for IT Infrastructure
+Automation & Compliance.
+
+See: http://rudder-project.org for more information.
+
+=== Synopsis
+
+This plugin provides fine grained Access Control List on APIs.
+It also allows registered Rudder users to get private token with the same
+rights as their role allows.
+
+=== Logging
+
+You can log information about ACL (behavior and errors) by adding the following lines in your ```logback.xml``` file:
+
+
+```
+
+  <!--
+      API ACLs
+      ========
+      Information about ALC evalutation for APIs.
+
+      This logger allows to get extra information about API
+      ACLs resolution.
+
+   -->
+  <logger name="api-acl" level="off" />
+```
+
+// Everything after this line goes into Rudder documentation
+// ====doc====
+
+
+=== User personnal API Token
```

Comment:
This should be a first level title "=", doc level integration is handle by the doc.